src/lib/components/display/tabs-links.svelte

```diff
@@ -1,14 +1,17 @@
 <script lang="ts">
-  export let items = [];
-  export let itemId = undefined;
+  import type { TabItem } from "$lib/types";
+
+  export let items: TabItem[] = [];
+  export let noScrollOnChange = false;
+  export let itemId: string | undefined = undefined;
```

Comment:
Il n'y a pas de raison de le rendre optionnel à mon avis.
```suggestion
  export let itemId: string;
```